pkg/cmd/cli/cmd/importimage.go

```diff
@@ -45,6 +45,7 @@ func NewCmdImportImage(fullName string, f *clientcmd.Factory, out io.Writer) *co
 	}
 	cmd.Flags().String("from", "", "A Docker image repository to import images from")
 	cmd.Flags().Bool("confirm", false, "If true, allow the image stream import location to be set or changed")
+	cmd.Flags().Bool("insecure-registry", false, "If true, allow the docker registry to be insecure")
```

Comment:
@liggitt asked this to reflect the annotation, which is `openshift.io/image.insecureRepository`, thus proposal for `insecure-repository, can you please change this?